### ISSUE TITLE
Angular 11 Sample not working

### ISSUE THREAD

**Issue by patrick-robin** [AzureAD/microsoft-authentication-library-for-js#2859]
## Library

- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-node@1.x.x`
- [ ] `@azure/msal-react@1.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [x] `@azure/msal-angular@2.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Framework

- [x] Angular
- [ ] React
- [ ] Other

## Description

I'm trying to run the _angular-11-sample-app_ but when I run _npm start_ I get a number of errors stating `Property 'setActiveAccount' does not exist on type 'IPublicClientApplication'.`

## Error Message

```
Error: src/app/home/home.component.ts:18:37 - error TS2339: Property 'setActiveAccount' does not exist on type 'IPublicClientApplication'.

18           this.authService.instance.setActiveAccount(result.account);
                                       ~~~~~~~~~~~~~~~~
src/app/app.component.ts:48:39 - error TS2339: Property 'setActiveAccount' does not exist on type 'IPublicClientApplication'.

48             this.authService.instance.setActiveAccount(response.account);
                                         ~~~~~~~~~~~~~~~~
src/app/app.component.ts:54:41 - error TS2339: Property 'setActiveAccount' does not exist on type 'IPublicClientApplication'.

54               this.authService.instance.setActiveAccount(response.account);
                                           ~~~~~~~~~~~~~~~~
src/app/app.module.ts:48:5 - error TS2322: Type '{ interactionType: InteractionType.Redirect; loginFailedRoute: string; }' is not assignable to type 'MsalGuardConfiguration'.
  Object literal may only specify known properties, and 'loginFailedRoute' does not exist in type 'MsalGuardConfiguration'.

48     loginFailedRoute: '/login-failed'
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps

- Checkout master branch
- Open _angular11-sample-app_ in VS Code
- Run _npm install_
- Run _npm start_

## Expected behavior

The application runs without any errors.

## Identity Provider

- [x] Azure AD
- [ ] Azure B2C Basic Policy
- [ ] Azure B2C Custom Policy
- [ ] ADFS
- [ ] Other

## Browsers/Environment

- [ ] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

## Regression

- [ ] Did this behavior work before?
Version:

## Security

- [ ] Is this issue security related?

## Source

- [ ] Internal (Microsoft)
- [x] Customer request

**Comment by web265p3** [comment]
Just wanted to open the same issue...

**Comment by jo-arroyo** [comment]
@patrick-robin @web265p3 The latest versions of msal-browser and msal-angular were released last night. Please update by `npm install @azure/msal-browser@2.9.0 @azure/msal-angular@alpha` and give the new version a try.

**Comment by zibeeze** [comment]
Thanks @jo-arroyo updated worked for me.

**Comment by Maraud3r** [comment]
@jo-arroyo I have question about the latest packages. I updated the Angular 11-Sample-App and it now compiles and runs.

I changed the client ID to be a app I created in my azure tenancy.

I can run the angular app, login and I can see in the console where it's displaying the info that it gets back.

What I'm confused about is the tenantId, it's not my tenantId.

I've searched through the angular code and I don't find a reference to a place that the tenant id is set.

Is the tenant id that's returned not the real tenant id?
